docs/sdk_developers/training/testing_forks.md

```
@@ -44,20 +44,25 @@ Now that your fork has the updated logic, you need to create an event to trigger
-If a bot waits for **21 days**, change the variable to **0 days** or **minutes** in your script.
+The contributor-lifecycle bot (`.github/workflows/bot-contributor-lifecycle.yml` →
+`.github/scripts/bot-contributor-lifecycle.js`) reads its thresholds from environment
+variables, so you do **not** need to edit any code — just trigger the workflow manually
+with the thresholds set to `0` (and `dry_run` off) so everything is treated as immediately
+stale:
 
-**Before (Production Code):**
-```bash
-# Wait for 21 days before marking as stale
-DAYS="${DAYS:-21}"
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)